source/features/wait-for-attachments.ts

```diff
@@ -0,0 +1,42 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+// `data-required-trimmed` overrides the `disabled` state of the form
+const attribute = 'data-required-trimmed';
+const attributeBackup = 'data-rgh-required-trimmed';
+
+function toggleSubmitButtons({target, type}: Event): void {
+	const form = target as HTMLFormElement;
+	const textarea = select('textarea', form)!;
```

Comment:
The selector can’t be simplified 019556fa9b269d0895ef071aa57cd3656f4e532f

Maybe it’s worth adding a comment to explain why the selector is complex.

Reply:
I restored the selector, and added a check to make the feature work on PR pages and when editing a comment.

Reply:
I added a comment to explain what exactly is happening here.

I haven't re-verified our code based on this explanation, but I think this feature should:

- temporarily remove `data-disable-invalid`
- permanently add `data-disable-invalid` to the buttons that don't have it

Does that sound correct? Is it true that we can (oops, once again) drop the loop and instead just permanently add `data-disable-invalid` to those buttons.

The only issue I see is finding the right time to add `data-disable-invalid`, maybe that should only happen in `upload:setup`

Reply:
> Does that sound correct? Is it true that we can (oops, once again) drop the loop and instead just permanently add data-disable-invalid to those buttons.

Indeed, we don't need to disable the buttons, adding the attribute works. Tested in the three cases (new comment, edit and new PR).
The loop could probably be optimized though, we don't need it to run every time.